qa/L0_pytorch_python_runtime/test.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
What do you think of moving this test into pytorch backend repo with `BACKEND:PYTORCH` child pipeline to keep things modular?

Reply:
Can you elaborate a bit more on your thought? We already have `PYTORCH:TEST` child pipeline and this test is one of the tests on the child pipeline. If we are going to move all the tests on `PYTORCH:TEST` child pipeline from server repo to pytorch_backend repo, maybe we can file another ticket for it?

Reply:
I meant `PYTORCH:TEST` and yes if it is being included in that child pipeline on gitlab side, then we're good for now.